components/ArticleList.jsx

```diff
@@ -0,0 +1,25 @@
+import { useEffect, useState } from "react";
+import ArticleCard from "./ArticleCard";
+import axios from "axios";
+
+const ArticleList = () => {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    axios.get("https://newssite-zy4v.onrender.com/api/articles").then((res) => {
+      setArticles(res.data.articles);
+    });
+  }, []);
+
```

Comment:
If the data takes a while to load, could you provide some visual feedback to let the user know?